src/lib/y2partitioner/widgets/help.rb

```diff
@@ -37,6 +37,22 @@ def included(_target)
         bios_id:                   N_("<b>BIOS ID</b> shows the BIOS ID of the hard\n" \
                                       "disk. This field can be empty."),
 
+        btrfs_exclusive:           N_("<b>Excl. Size</b> shows the size of the exclusive space\n" \
+                                      "of the BtrFS subvolume. This is the space that would be\n" \
+                                      "freed by deleting the subvolume. Known and displayed only\n" \
+                                      "if quotas were active for the corresponding BtrFS file\n" \
+                                      "system during the hardware probing."),
```

Comment:
NP: maybe we should avoid to use *probing*. What about *during the hardware detection*?

Reply:
Done